stacks-signer/src/v0/signer.rs

```diff
@@ -607,16 +607,12 @@ impl Signer {
         };
 
         // authenticate the signature -- it must be signed by one of the stacking set
-        let is_valid_sig = self
-            .signer_addresses
-            .iter()
-            .find(|addr| {
-                let stacker_address = StacksAddress::p2pkh(true, &public_key);
+        let is_valid_sig = self.signer_addresses.iter().any(|addr| {
+            let stacker_address = StacksAddress::p2pkh(true, &public_key);
```

Comment:
I forget, doesn't this `is_mainnet` flag need to be dynamic? I can't remember right now if this set of addresses is fixed to mainnet or not.

Reply:
woops yes it should be.